manim/utils/color/core.py

```diff
@@ -606,6 +606,92 @@ def interpolate(self, other: ManimColor, alpha: float) -> Self:
             self._internal_space * (1 - alpha) + other._internal_space * alpha
         )
 
+    def darker(self, factor: float = 0.2) -> Self:
+        """Returns a new color that is darker than the current color, i.e.
+        interpolated with black. The opacity is unchanged.
+
+        Parameters
+        ----------
+        factor : float, optional
+            The factor by which the color should be darker, by default 0.2,
+            which results in a slightly darker color
```

Comment:
I renamed the argument to `blend`, as I was not very happy with `factor` which is not very descriptive. I changed the wording of the help text to include the terms "blend ratio" and indicate what 0 and 1 mean. I just don't think adding "float" is necessary, as it's already indicated that the argument has type float in both the type hint and the docstring, and in any case it should be evident since it's between 0 and 1

Reply:
Also I see you've added a period at the end, but it seems the majority of other docstrings don't have them, at least in this module. Is it preferred to add one?

Reply:
`blend` is a great choice, and the parameter descriptions read very well now.

When I reviewed this in the first round, I confused our usual style rules for oneline descriptions (which usually do end with a full stop) and the rules we use (although much less consistently) for parameter descriptions. It was actually more correct the way you had it initially.

Sorry for the confusion (I was sort of in a hurry when I reviewed this; unfortunately this was quite noticeable this time ;-)).